<launch>
    <node pkg="rosbag" type="play" name="rosbag" required="true" args="-s 20 $(find camera_calibration)/../../../data/2016-11-22-14-32-13_test.bag"/>
    <node name="imag_rect" pkg="image_proc" type="image_proc">
        <remap from="/image_raw" to="/sensors/camera/image_color"/>
        <remap from="/camera_info" to="/sensors/camera/camera_info"/>
    </node>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find camera_calibration)/launch/rviz_config.rviz"/>

    <node pkg="camera_calibration" type="overlapScansOnImages_node" name="overlapScansOnImages">
        <remap from="/sensors/velodyne_points" to="/CheckerBoardSurface"/>
    </node>
    <node pkg="camera_calibration" type="preProcessPC_node" name="preProcess"/>
</launch>
